doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -3442,10 +3442,60 @@ Archiving is an optional feature that may be configured for a Dataverse installa
 
   curl -H "X-Dataverse-key: $API_TOKEN" -X DELETE "$SERVER_URL/api/datasets/:persistentId/$VERSION/archivalStatus?persistentId=$PERSISTENT_IDENTIFIER"
   
+Get Dataset External Tool URL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This API call generates a URL for accessing an external tool (see :doc:`/installation/external-tools`) that operates at the dataset level. The URL includes necessary authentication tokens and parameters based on the user's permissions and the tool's configuration.
+
+Authentication is required for draft or deaccessioned datasets and the user must have ViewUnpublishedDataset permission.
+
+.. code-block:: bash
+
+  export API_TOKEN=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+  export SERVER_URL=https://demo.dataverse.org
+  export DATASET_ID=1234
```

Comment:
For this example, it might be nice to show a PID instead of a database ID for a dataset. (I assume it works. I haven't tested it.) People are more likely to know the PID.